webui/react/src/components/WorkspaceCreateModal.tsx

```diff
@@ -110,6 +167,33 @@ const WorkspaceCreateModalComponent: React.FC<Props> = ({ onClose, workspaceId }
           ]}>
           <Input maxLength={80} />
         </Form.Item>
+        {canModifyBindings && resourceManagers.length > 0 && (
+          <>
+            <Divider />
+            <>Namespace Bindings:</>
+            <br />
+            <div style={{ color: '#8a8888' }}>
+              Note: If you leave the Namespace name blank, the workspace will be bound to the
+              default Namespace configured in the Master Config.
+            </div>
+            <br />
+            <>
+              {resourceManagers.map((name) => (
+                <Form.Item
+                  key={name}
+                  label={`${name}`}
+                  name={`${name}`}
+                  rules={[
+                    {
+                      required: false,
+                    },
+                  ]}>
+                  <Input maxLength={63} />
+                </Form.Item>
+              ))}
+            </>
```

Comment:
see above re: unnecessary fragment. in addition, we should be able to directly pass the names to the label and name props without string interpolation and form items should default to not being required.
```suggestion
            {resourceManagers.map((name) => (
              <Form.Item
                key={name}
                label={name}
                name={name}
              >
                <Input maxLength={63} />
              </Form.Item>
            ))}
```

Reply:
done!